src/webui.ts

```diff
@@ -255,10 +264,10 @@ export function run(win: Usize, script: string): boolean {
   return Boolean(status);
 }
 
-export function bind(
+export function bind<T extends string | number | boolean | undefined | void>(
```

Comment:
Is this fine?

Reply:
I read every line in your PR, and I agree with everything except one line. I don't know if it's correct.
Please correct me if I'm wrong, but `func` should only refer to a callback function, so `string`, `number`, `boolean`, and `undefined` won't work correctly, right?

Reply:
The value return by the callback is send to the client when calling "webui_fn" global function. As it use underlaying websocket it can normally pass any primitives or buffer/blob but the c lib seems to only allow string, number and boolean. In fact I've not tested it.

Reply:
Actual bin implementation cast all input to C-String, I will check if some other primitives can be allowed without type issues.